packages/react-components/src/components/chart/store/context.tsx

```diff
@@ -10,7 +10,7 @@ import useDataStore from '../../../store';
 // Recommended usage from zustand https://docs.pmnd.rs/zustand/previous-versions/zustand-v3-create-context
 export const ChartStoreContext = createContext<
   ReturnType<typeof createChartStore>
->(createChartStore());
+>(createChartStore({ chartId: 'contextStore' }));
```

Comment:
is there a way to indicate to the persist middleware not to persist something? This store should actually never be used, we only create the reference here because typescript would complain otherwise. I think if it's possible, we shouldn't save it since it would just be taking up space best case and a potential for bugs worst case.

Reply:
Maybe it's as simple as persisting the store only if an id is provided to the `createChartStore`? and just doing a ternary in the function?

Reply:
Also, maybe instead of actually creating the store reference in the context, we should just use null/undefined as the initial value, and then use invariants to safeguard and not have to null coalesce everything. I would have to play around with that to see if it's actually doable without a big structural change, so don't feel like you need to make that change, just another thought!

Reply:
Setting it to null makes sense since that is technically the initial value of any store.

Reply:
Made a ticket for this, so this gets in and removes the weird highlight/hidden persistence issues